token/program2/src/instruction.rs

```diff
@@ -121,34 +122,37 @@ pub enum TokenInstruction {
     ///   1. '[]' The source account's multisignature owner.
```

Comment:
Shoudl `s/owner/authority/` ?

Reply:
An account's authority is still called `owner`, so I decided to leave that term in the docs.